gsoc2018-bharat/src/create_db.py

```diff
@@ -0,0 +1,56 @@
+import json
+from gensim.models import FastText
+import torch
+from torch.autograd import Variable
+import numpy as np
+import Encoder
+import logging
+
+
+logging.basicConfig(format='%(levelname)s : %(message)s', level=logging.INFO)
+logging.root.level = logging.INFO
+model = torch.load('model/description_encoder')
+m = FastText.load('model/entity_fasttext_n100')
+wv = m.wv
+del m
```

Comment:
Is the creation of the `m` object necessary?

Reply:
Yes. I need to load all the pre-trained embeddings into the main memory. I delete it because I only need the keyed vectors, not the entire model. I can make use of GloVe instead of FastText to save the hassle of loading the model first.